shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -171,6 +171,24 @@
   return active_composition_order_ != composition_order_;
 }
 
+const int FlutterPlatformViewsController::kDefaultMergedLeaseDuration;
+
+bool FlutterPlatformViewsController::PostPrerollAction(
+    fml::RefPtr<fml::GpuThreadMerger> gpu_thread_merger) {
+  const bool uiviews_mutated = HasPendingViewOperations();
+  if (uiviews_mutated) {
+    bool are_merged = gpu_thread_merger->IsMerged();
```

Comment:
nit: `is_merged`

Reply:
The name is also clean enough now that I'm not sure the stack variable is needed

Reply:
Done